tests/unittests/qml/tst_ChromeController.qml

```diff
@@ -175,69 +172,35 @@ ChromeController {
+            webviewMock.isFullScreen = false
             compare(webviewMock.locationBarController.mode, data.mode)
             compare(showSpy.count, data.shown ? 1 : 0)
         }
 
-        function test_loading_state_changed_data() {
-            return [
-                {forceHide: false, forceShow: false, fullscreen: false,
-                 mode: modeAuto, shown: true},
-                {forceHide: false, forceShow: false, fullscreen: false,
-                 mode: modeShown, shown: false},
-                {forceHide: false, forceShow: false, fullscreen: false,
-                 mode: modeHidden, shown: false},
-                {forceHide: true, forceShow: false, fullscreen: false,
-                 mode: modeHidden, shown: false},
-                {forceHide: false, forceShow: true, fullscreen: false,
-                 mode: modeShown, shown: false},
-                {forceHide: false, forceShow: false, fullscreen: true,
-                 mode: modeHidden, shown: false},
-                {forceHide: true, forceShow: true, fullscreen: false,
-                 mode: modeHidden, shown: false},
-                {forceHide: true, forceShow: false, fullscreen: true,
-                 mode: modeHidden, shown: false},
-                {forceHide: false, forceShow: true, fullscreen: true,
-                 mode: modeShown, shown: false},
-                {forceHide: true, forceShow: true, fullscreen: true,
-                 mode: modeHidden, shown: false},
-            ]
-        }
-
-        function test_loading_state_changed(data) {
-            controller.forceHide = data.forceHide
-            controller.forceShow = data.forceShow
-            webviewMock.fullscreen = data.fullscreen
-            webviewMock.locationBarController.mode = data.mode
-            showSpy.clear()
-            webviewMock.loading = true
-            compare(showSpy.count, data.shown ? 1 : 0)
-            compare(webviewMock.locationBarController.mode, data.mode)
-            showSpy.clear()
-            webviewMock.loading = false
-            compare(showSpy.count, 0)
-            compare(webviewMock.locationBarController.mode, data.mode)
-        }
-
         function test_load_event_data() {
             var data = []
-            var booleanValues = [false, true]
-            var modeValues = [modeAuto, modeHidden, modeShown]
-            for (var i in booleanValues) {
-                for (var j in booleanValues) {
-                    for (var k in modeValues) {
-                        for (var l in modeValues) {
-                            data.push({forceHide: booleanValues[i], forceShow: booleanValues[j],
-                                       initialMode: modeValues[k], defaultMode: modeValues[l]})
+            var forceHideValues = [false, true];
+            var forceShowValues = [false, true];
+            var initialḾodeValues = [modeAuto, modeHidden, modeShown];
+            var defaultḾodeValues = [modeAuto, modeHidden, modeShown];
```

Comment:
How did you get accented M's?

Reply:
my keyboard is not short of surprises : )